crates/llvm-context/src/polkavm/context/mod.rs

```diff
@@ -698,17 +686,13 @@ where
                     revive_runtime_api::polkavm_imports::GET_STORAGE,
                     &[
                         self.xlen_type().const_int(transient as u64, false).into(),
-                        storage_key_pointer_casted.into(),
+                        pointer.to_int(self).into(),
                         self.xlen_type().const_all_ones().into(),
                         storage_value_pointer.to_int(self).into(),
                         storage_value_length_pointer.to_int(self).into(),
                     ],
                 );
 
-                // We do not to check the return value.
```

Comment:
The comment is still valid though, could we leave it in place here?

Reply:
was removed when I added the check 😅